model_analyzer/config/generate/run_config_generator_factory.py

```diff
@@ -92,7 +92,7 @@ def _create_search_config(command_config):
 
         search_config = SearchConfig(dimensions=dimensions,
                                      radius=3,
-                                     step_magnitude=20,
+                                     step_magnitude=3,
```

Comment:
Remove magic numbers and replace with defines

Reply:
Should we define these under `constants.py` since these aren't going to be specified through config (at least for now)?

Reply:
Yes, that would be my preference